flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackendBuilder.java

```diff
@@ -238,6 +242,16 @@ public RocksDBKeyedStateBackendBuilder(
         return this;
     }
 
+    RocksDBKeyedStateBackendBuilder<K> setRocksDBStateUploader(
+            RocksDBStateUploader rocksDBStateUploader) {
+        Preconditions.checkState(
```

Comment:
```suggestion
        Preconditions.checkState(
                injectRocksDBStateUploader == null, "rocksDBStateUploader can be only set once");
        Preconditions.checkState(
```
I actually meant that the `rocksDBStateUploader` should be `null`, but checking `numberOfTransferingThreads` also makes sense)

Reply:
thx very much . I also think it would be better to add this check